packages/core/admin/admin/src/content-manager/components/FieldTypeIcon/index.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Box } from '@strapi/design-system/Box';
+import { useCustomFields } from '@strapi/helper-plugin';
```

Comment:
any reason it's in the helper-plugin ? It looks like it is a core admin hook to me

Reply:
Ok This is something I think we need to refactor but that can wait